.github/workflows/preview_on_auspice_us.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Preview on auspice.us"
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+
+  workflow_dispatch:
+
+jobs:
+  run:
+    # Run if PR has the label and is updated or is newly labeled with the label
+    if: |
+      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'preview on auspice.us')) ||
+      (github.event.action == 'labeled' && github.event.label.name == 'preview on auspice.us')
```

Comment:
I believe this logic prevents runs from `workflow_dispatch` events, should we just remove the `workflow_dispatch` trigger? (I just tried a [manual run on master](https://github.com/nextstrain/auspice/actions/runs/8694430624) and the job was skipped).

Reply:
Oh yes, not sure why I copied it. Removing in #1768